script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -387,17 +403,22 @@ After raising her brow, Akiha stands up without making a sound.
 [sha:0f5602b824bb6e7e63de7e24e0332f15877cb5a6]{
 -- Page 86, Offset 19886.
 -- てっきり冷酷に切り返してくるかと思ったが、秋葉は逆に、気まずそうに視線を逸らした。
-I thought for sure that she would ruthlessly strike back at me, but on the contrary, Akiha looks away with an unpleasant expression.
+I thought that she'd have a ruthless counterattack in store for me,
+ but contrary to my expectations Akiha turns away,
+ an uneasy expression on her face.
 }
 [sha:98eec20701d7764ad76fd457acf3f534635efb58]{
 -- Page 87, Offset 19887.
 -- 「私だって、こんな事は言いたくありません。
-"Even I didn't want to have a conversation like this.
+"It's not as though I wanted to have this conversation either.
```

Comment:
It's not like I wanted to have this conversation either.

Reply:
Somewhat prefer `as though` to `like` for Akiha since it reads as more refined to me

Reply:
In most cases I'd agree, but this one just reads a bit stilted imo.